specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -608,6 +777,69 @@
           "type": "string"
         }
       }
+    },
+    "ChangeDirectoryRequest": {
+      "description": "The parameters required to perform change directory operation.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ChangeDirectoryProperties"
+        }
+      }
+    },
+    "ChangeDirectoryProperties": {
+      "description": "The parameters required to perform change directory operation",
+      "required": [
+        "destinationEmailId"
+      ],
+      "properties": {
+        "destinationEmailId": {
+          "description": "Email Id of the destination user",
+          "type": "string"
+        }
+      }
+    },
+    "ChangeDirectoryResponseProperties": {
+      "description": "The response body for change directory operation",
+      "properties": {
+        "destinationEmailId": {
+          "description": "Email Id of the destination user",
+          "type": "string"
+        },
+        "status": {
```

Comment:
make it enum and call out all the possible values.

Reply:
Added enum